tests/src/main/java/testcases/T052_binds_interface_generic/A.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package testcases.T052_binds_interface_generic;
+
+public class A<T> implements B<T> {}
```

Comment:
What happens with wildcards?

Reply:
Will add tests

Reply:
Wildcards aren't valid here - what do you mean?